src/Escape/EscapeInt.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
```

Comment:
There are no unit tests for these classes 🤔

Reply:
It is covered by more high-level tests, but as I said before, I think whole espacing will be done differently.